src/react-server-dom-webpack/cjs/react-server-dom-webpack-client.node.unbundled.production.js

```
@@ -11,7 +11,8 @@
 "use strict";
 var util = require("util"),
   ReactDOM = require("react-dom"),
-  decoderOptions = { stream: !0 };
+  decoderOptions = { stream: !0 },
+  hasOwnProperty = Object.prototype.hasOwnProperty;
```

Biome (2.1.2)

[error] Do not shadow the global "hasOwnProperty" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)